pkg/distro/rhel9/edge.go

```diff
@@ -24,8 +24,9 @@ var (
 			osPkgsKey: edgeCommitPackageSet,
 		},
 		defaultImageConfig: &distro.ImageConfig{
-			EnabledServices: edgeServices,
-			SystemdUnit:     systemdUnits,
+			EnabledServices:       edgeServices,
+			SystemdUnit:           systemdUnits,
+			CreateUsersSodoerFile: common.ToPtr(true),
```

Comment:
I see you've enabled the option for the edge-commit (both here and in RHEL 8).  I'm curious if that's what we want for functional parity with the current edge builds.  Afaik, Edge Fleet Management doesn't build edge commits with users in them but instead adds the user during installation using the kickstart.  So user configurations go into the `edge-installer` build.  If we stay with that workflow, this wont have any effect because no users will be added during the commit build.

We've been trying to get users to not create user accounts in the base ostree commits, because they become harder to manage.  If we're going to change this, we should figure out all the implications and guidance (upgrade commits for example need to include the same user otherwise it will be removed).

If. however, we want to stay to the current flow as much as possible (no users in commits, user creation in the kickstart), then this option (and the associated first-boot service creation) should happen in the ISO, using the kickstart.

I realise I didn't catch this earlier when we were discussing this solution.  I only realised what we were doing now when I tried to test the change myself.

Reply:
done via AnacondaInstallerISOTree as suggested.